packages/usa-tooltip/src/index.js

```diff
@@ -388,6 +388,16 @@ const tooltip = behavior(
     init(root) {
       selectOrMatches(TOOLTIP, root).forEach((tooltipTrigger) => {
         setUpAttributes(tooltipTrigger);
+
+        const { body, wrapper } = getTooltipElements(tooltipTrigger);
+        wrapper.addEventListener("mouseleave", () => {
+          hideToolTip(body);
+        });
+      });
+    },
+    teardown(root) {
+      selectOrMatches(TOOLTIP, root).forEach((tooltipWrapper) => {
+        tooltipWrapper.removeEventListener("mouseleave", hideToolTip);
```

Comment:
I'm not sure if this teardown is removing the event listener from the right element. It looks like the `forEach` is returning the tooltip trigger based on init.

 Should we rename `tooltipWrapper` to `tooltipTrigger` to match init, then get the element using `getTooltipElements`?

```suggestion
    teardown(root) {
      selectOrMatches(TOOLTIP, root).forEach((tooltipTrigger) => {
        const { wrapper } = getTooltipElements(toltipTrigger)
        wrapper.removeEventListener("mouseleave", hideToolTip);
```

Reply:
I changed it to `tooltipWrapper` here because (at least in my understanding) `TOOLTIP` has already been transformed into its "wrapper" state now that it has been initialized. So when it looks for `TOOLTIP` here, I believe it is looking for the initialized wrapper version of `usa-tooltip` rather than the original trigger version of `usa-tooltip`. Let me know if that doesn't make sense!

Reply:
Ahh yes! Ok that makes sense now considering the lifecycle of the component. Great observation! I'll go ahead and resolve this comment 👍